common/changes/@uifabric/api-docs/6.0_2019-07-23-19-58.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
All these change type none files shouldn't be needed. I think you may have something set up incorrectly which is causing Rush to think they're needed--hopefully that's something your mentor should be able to help resolve, or if not I can try and help.